controllers/pokemon.go

```diff
@@ -0,0 +1,65 @@
+package controllers
+
+import (
+	"Users/francisco.zamudio/projects/academy-go-q12021/repository"
+	"Users/francisco.zamudio/projects/academy-go-q12021/services"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+//HandlerGetPokemonByID Connects to a csv repository, then calls GetPokemonByID service
+//if correct it return the corresponding pokemon, else it return an error
+func HandlerGetPokemonByID(w http.ResponseWriter, r *http.Request) {
+
+	vars := mux.Vars(r)
+	dataRepository := repository.DataRepository{}
+	pokemonInteractor := services.PokemonInteractor{Repository: dataRepository}
+	pokemon, err := pokemonInteractor.GetPokemonByID(vars["id"])
+	if err != nil {
+		fmt.Println("An error encountered ::", err)
+		http.Error(w, err.Error(), 500)
```

Comment:
You already are importing net/http package, so you can use `http.StatusInternalServerError`

Reply:
done